bazel/fuzz_target.bzl

```diff
@@ -28,8 +28,14 @@ def java_fuzz_target_test(
         env = None,
         verify_crash_input = True,
         verify_crash_reproducer = True,
-        execute_crash_reproducer = False,
+        execute_crash_reproducer = True,
+        # Can't default to the empty string as it is passed as an argument and Bazel tokenizes the
+        # args.
+        expected_finding = "none",
         **kwargs):
+    if not verify_crash_reproducer:
```

Comment:
Create an error when `execute_crash_reproducer` is set but no `expected_finding` is given?